chains/migrations/0002_add_near_chain.py

```
@@ -0,0 +1,25 @@
+# Generated by Django 5.0.4 on 2024-07-15 18:24
+
+from django.db import migrations
+
+
+def create_near_chain(apps, schema_editor):
+    Chain = apps.get_model("chains", "Chain")
+    # Create the "near" chain
+    near_chain, created = Chain.objects.get_or_create(
```

Ruff (0.14.1)

Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)